codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/EventStreamTestTools.kt

```diff
@@ -100,8 +98,8 @@ object EventStreamTestTools {
 
         test.project.lib {
             when (variety) {
-                EventStreamTestVariety.Marshall -> writeMarshallTestCases(testCase, generator)
-                EventStreamTestVariety.Unmarshall -> writeUnmarshallTestCases(testCase, codegenTarget, generator)
+                EventStreamTestVariety.Marshall -> Unit // writeMarshallTestCases(testCase, generator)
+                EventStreamTestVariety.Unmarshall -> Unit // writeUnmarshallTestCases(testCase, codegenTarget, generator)
```

Comment:
Intentional?

Reply:
Yep, this file is removed was 2bc6b3f1bb12a3aeaa87fd1d9f45f0d2c346f276.